src/Outline/OutlineResultPrinter.php

```diff
@@ -111,6 +123,21 @@ protected function getResultText( QueryResult $res, $outputMode ) {
 			$result .= $link->getText( $outputMode, $this->mLinker ) . "\n";
 		}
 
+
+		// Add introtemplate
```

Comment:
Comments like this have downsides, see something like Clean Code by Robert C Martin.